wiki/Community/How_you_can_help!/it.md

```diff
@@ -0,0 +1,85 @@
+
+- Scegli un forum che suscita il tuo interesse, es.:
+  - [Help](https://osu.ppy.sh/community/forums/5)
+  - [Mapping Discussion](https://osu.ppy.sh/community/forums/56)
+  - [Storyboarding](https://osu.ppy.sh/community/forums/20)
+  - [Skinning](https://osu.ppy.sh/community/forums/15)
+  - [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13)
+- Leggi i topic fissati per imparare le regole di quel specifico forum e tutte le info utili che potrebbero contenere.
+- Usa il bottone `Segna il forum come già letto` in cima.
+- Segui qualsiasi nuovo thread, leggi ognuno di essi e aiuta quando puoi — puoi usare le icone non letto per assicurarti di rimanere aggiornato.
+- Leggi le [guide disponibili](/wiki/Guides) per migliorare la tua conoscenza di osu!. Se le guide non ti soddisfano, puoi provare a contribuire!
+
+## Aiutaci a controllare/verificare le beatmaps!
+
+osu! ha un team dedicato alla revisione, i [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators), per controllare che le [beatmaps](/wiki/Beatmap) inviate siano pronte ad essere classificate per essere giocate da tutti. Tuttavia, questo non vuol dire che non puoi contribuire in questo settore:
+
+- Guardate o cercate mappe [In Attesa](https://osu.ppy.sh/beatmapsets?s=pending) contenenti artisti o canzoni che ti piacciono e/o a cui vuoi contribuire.
+- Gioca beatmaps nella categoria [*In Attesa* or *Work-in-progress*](/wiki/Beatmap/Category#work-in-progress-and-pending) e lasciate dei feedback nella sezioni discussioni della beatmap.
+- Leggi la [Guida al Modding](/wiki/Modding) e i [Criteri per il Ranking](/wiki/Ranking_Criteria) per conoscere le basi del modding tecnico e cosa rende una beatmap classificabile.
+- Unisciti ai canali  `#mapping` e `#modreqs` nella chat in gioco e unisciti al discorso o semplicamente leggi cosa viene scritto.
+- Se hai tempo e dedizione, apri una [Modding Queue](https://osu.ppy.sh/community/forums/60) per dare uno spazio ai creatori di beatmap un posto per richiedere delle mods.
+
+## Dai feedback sui progressi dei giocatori o sulle skins!
+
+Tutti amano migliorarsi e necessitano di feedback. Spesso gli utenti chiedono feedback sui loro progressi nel gameplay nel [forum Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) e altri attendono un feedback sulle loro ultime skin create nel [forum Work In Progress Skins](https://osu.ppy.sh/community/forums/119).
+
+## Aiuta o unisciti a progetti gestiti dagli utenti
+
+osu! ha una comunità fiorente di persone tecniche e competitive. Variano da hobbisti a veterani dei rhythm games. Ci sono molte proposte e progetti sviluppati esclusivamente per osu! e tu puoi aiutarli, o guidare il futuro di osu! con le tue proposte/progetti!
+
+Ecco alcuni dei modi con cui puoi contribuire:
+
+- Sfoglia il forum [Tornei](https://osu.ppy.sh/community/forums/55). I tornei richiedono molto tempo e lavoro per organizzarli senza alcun intoppo, tu puoi aiutarli facendo da giudici spettatori in certi casi.
```

Comment:
The second part of the phrase looks odd to me. I would do something like:
"I tornei richiedono molto tempo e lavoro per essere organizzati senza alcun intoppo. Puoi aiutare facendo da giudice spettatore in determinati casi."